plugins/test/unit/plugins/distributors/yum/test_configuration.py

```diff
@@ -680,16 +675,14 @@ def test_get_repo_checksum_update_distributor_config_non_yum(self, mock_distribu
             {SCRATCHPAD_DEFAULT_METADATA_CHECKSUM: 'sha1'}
         self.assertEquals('sha1',
                           configuration.get_repo_checksum_type(self.mock_conduit, self.config))
-        self.assertFalse(mock_distributor_manager.return_value.update_distributor_config.called)
+        self.assertFalse(m_dist_ctrl.update.called)
```

Comment:
I think we should make a stronger assertion here, but it can be done as a later PR since this is no different than the assertion that was already present.